zmon_aws_agent/postgresql.py

```diff
@@ -92,6 +91,23 @@ def collect_instances(infrastructure_account):
     return [i['Instances'][0] for i in instances if i['OwnerId'] == infrastructure_account.split(':')[1]]
 
 
+def extract_eip_allocation_from_lc(infrastructure_account, cluster_name):
+    import yaml
+    import base64
+
+    asg = boto3.client('autoscaling')
+    lc_paginator = asg.get_paginator('describe_launch_configurations')
+    lcs = lc_paginator.paginate().build_full_result()['LaunchConfigurations']
```

Comment:
It needs to be cached somewhere, because for every cluster it will call `describe_launch_configurations` and we already have rate limit problem very often.